test/deployment/AptTest.java

```diff
@@ -38,9 +38,9 @@
 
 public class AptTest {
     private static final Logger LOG = LoggerFactory.getLogger(AptTest.class);
-    private static final String aptSnapshot = "https://repo.vaticle.com/repository/apt-snapshot/";
-    private static final String aptRelease = "https://repo.vaticle.com/repository/apt/";
-    private static final String pubkey1 = "8F3DA4B5E9AEF44C";
+    private static final String aptSnapshot = "https://repo.typedb.com/public/public-snapshot/deb/ubuntu";
+    private static final String aptRelease = "https://repo.typedb.com/public/public-release/deb/ubuntu";
+    private static final String pubkey1 = "17507562824cfdcc";
     private static final String pubkey2 = "https://cli-assets.heroku.com/apt/release.key";
     private static final String pubkey3 = "https://dl.google.com/linux/linux_signing_key.pub";
```

Comment:
For post-release: what are these keys/what are these used for? Do we have to update them at all?